### PR TITLE
Splitting prepare-testing-bucket output to be per server type

### PR DESCRIPTION
**Pull request by kobymeir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Splitting prepare-testing-bucket output to be per server type

https://github.com/demisto/content-private/pull/692

### CODE REVIEW

File: .gitlab/ci/.gitlab-ci.variables.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ variables:
   IFRA_ENV_TYPE: "Content-Env"
   GIT_DEPTH: 0
   GET_SOURCES_ATTEMPTS: 3  # see https://docs.gitlab.com/ee/ci/runners/configure_runners.html#job-stages-attempts
-  ENV_RESULTS_PATH: "${CI_PROJECT_DIR}/artifacts/env_results.json"
+# FIXME! remove  ENV_RESULTS_PATH: "${CI_PROJECT_DIR}/artifacts/env_results.json"
```
Comment (kobymeir) [inline .gitlab/ci/.gitlab-ci.variables.yml:31]: will be removed before merge

File: Tests/scripts/collect_tests/path_manager.py [hunk 2]
```diff
@@ -22,7 +22,8 @@ class PathManager:
     """
     Used for getting paths of various files and folders during the test collection process.
     """
-    ARTIFACTS_PATH = Path(os.getenv('ARTIFACTS_FOLDER', './artifacts'))
+    # FIXME! remove ARTIFACTS_PATH = Path(os.getenv('ARTIFACTS_FOLDER', './artifacts'))
```
Comment (kobymeir) [inline Tests/scripts/collect_tests/path_manager.py:25]: will be removed before merge

**Review by omerKarkKatz — COMMENTED** [review]
Left a comment relating to content private

File: .gitlab/ci/.gitlab-ci.variables.yml (hunk 1 above)
Comment (kobymeir) [inline .gitlab/ci/.gitlab-ci.variables.yml:31]:
```suggestion
```

File: Tests/scripts/collect_tests/path_manager.py (hunk 2 above)
Comment (kobymeir) [inline Tests/scripts/collect_tests/path_manager.py:25]:
```suggestion
```

**Review by omerKarkKatz — APPROVED** [review]
Good job!

**Pull request merged**